cardano-node/src/Cardano/Tracing/Config.hs

```diff
@@ -70,7 +70,7 @@ type TraceBlockFetchDecisions = ("TraceBlockFetchDecisions" :: Symbol)
 type TraceBlockFetchProtocol = ("TraceBlockFetchProtocol" :: Symbol)
 type TraceBlockFetchProtocolSerialised = ("TraceBlockFetchProtocolSerialised" :: Symbol)
 type TraceBlockFetchServer = ("TraceBlockFetchServer" :: Symbol)
-type TraceChainDB = ("TraceChainDB" :: Symbol)
+type TraceChainDB = ("TraceChainDb" :: Symbol)
```

Comment:
FYI, I don't know the consequences of this. Perhaps someone else should review this line.

Reply:
In particular, in the JSON configuration provided in `configuration/cardano/mainnet-config.json` it is written as `"TraceChainDb": true` and without this fix it was not being picked up because it was expecting `"TraceChainDB"` as @jutaro mentions

Reply:
Nice catch @jutaro